python/paddle/fluid/layers/math_op_patch.py

```diff
@@ -370,13 +370,7 @@ def __impl__(self, other_var):
             setattr(Variable, method_name, method_impl)
     else:
         import paddle.tensor
-        variabel_methods = paddle.tensor.linalg.__all__ + \
-                           paddle.tensor.math.__all__ + \
-                           paddle.tensor.logic.__all__ + \
-                           paddle.tensor.manipulation.__all__ + \
-                           paddle.tensor.search.__all__ + \
-                           paddle.tensor.stat.__all__ + \
-                           paddle.tensor.attribute.__all__
+        variabel_methods = paddle.tensor.binary_ops
```

Comment:
only sume of the methods of paddle.Tensor is created by `_binary_creator_`, other methods are just the independent functions with the same name. 

These may not be binary methods.

I suggest changing the name.

Reply:
done, binary_ops->tensor_method_func